vllm/attention/backends/hpu_attn.py

```diff
@@ -8,7 +8,8 @@
 
 import torch
 import vllm_hpu_extension.ops as ops
-from vllm_hpu_extension.utils import Matmul, Softmax, VLLMKVCache
+from vllm_hpu_extension.utils import (Matmul, ModuleFusedSDPA, Softmax,
```

Comment:
please update requirements-hpu file to point to vllm extension @ 41ff369

Reply:
Done